content/data/unicorns.json

```
@@ -439,5 +439,24 @@
       "author",
       "translator"
     ]
+  },
+  {
+    "id": "williamcook",
+    "name": "William George Cook",
+    "firstName": "William",
+    "lastName": "Cook",
+    "description": "Full stack developer who loves to help! Find me in the garden, up a tree, or at Disney World ✨",
+    "socials": {
+      "twitter": "wgeorgecook",
+      "github": "wgeorgecook",
+      "linkedin": "wgeorgecook",
+      "website": "https://williamgeorgecook.com"
+    },
+    "pronouns": "he",
+    "profileImg": "./williamcook.jpg",
+    "color": "#AF7AC5",
+    "roles": [
+      "developer"
```

This role is actually reserved for developers of the Unicorn Utterances site itself 😅 Instead, let's do:

```suggestion
      "author"
```